pyarn/lockfile.py

```diff
@@ -88,3 +91,50 @@ def from_str(cls, lockfile_str):
             if declared_version:
                 version = declared_version.group(1)
         return cls(version, parsed_data['data'])
+
+    def to_file(self, path):
+        with open(path, 'w') as lockfile:
+            self._dump(lockfile)
+
+    def to_str(self):
+        buffer = io.StringIO()
+        self._dump(buffer)
+        return buffer.getvalue()
+
+    def _dump(self, outfile):
+        # Does not preserve any comments, but this one is required
+        outfile.write('# yarn lockfile v1\n')
```

Comment:
Should this be a constant somewhere?

I realized later after processing this string that it is only used for lockfile v1.

Reply:
Should the constant also be used when determining the file version instead of the regex?

Reply:
Sounds good, yes!

Reply:
ok, changed :-)